pkgs/development/libraries/science/math/scs/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, blas, liblapack, gfortran }:
+
+stdenv.mkDerivation rec {
+  name = "scs-${version}";
+  version = "2.0.2";
+
+  src = fetchFromGitHub {
+    owner = "cvxgrp";
+    repo = "scs";
+    rev = "v${version}";
+    sha256 = "17lbcmcsniqlyzgbzmjipfd0rrk25a8hzh7l5wl2wp1iwsd8c3a9";
+  };
+
+  buildInputs = [ blas liblapack gfortran.cc.lib ];
+
+  # Actually link and add libgfortran to the rpath
+  patchPhase = ''
+    sed -i 's/#-lgfortran/-lgfortran/' scs.mk
+  '';
+
+  doCheck = true;
+  checkPhase = ''
+    ./out/demo_socp_indirect 42
+  '';
+
```

Comment:
Upstream makefile doesn't provide standard check/install targets, but I think these get the job done.